File: Algoritmos Traduzidos/Lua/Algoritmos(FilipeDias)/ex8.lua
m=4

io.write'Insira o peso dos peixes:'

peso = io.read()


if tonumber(peso)>50 then

	ex=peso-50
	ex=ex*m

	io.write('Tem de pagar', ex, 'euros pelo excedente de peixe apanhado')

else

	print'Nao precisa de pagar multa'

end

File: Algoritmos Traduzidos/Python/Algoritmos[FilipeDias]/ex8.py
m=4

peso=input('Insira o peso dos peixes:')

if int(peso)>50:

    ex=int(peso)-50
    ex=ex*m

    print('Tem de pagar',ex,'euros pelo excedente de peixe apanhado')

else:

    print('Não precisa de pagar multa')
    
